hearthstone/entities.py

```diff
@@ -70,6 +71,8 @@ def register_entity(self, entity):
 		self.entities.append(entity)
 		if isinstance(entity, Player):
 			self.players.append(entity)
+		if not self.setup_done:
+			self.initial_entities.append(entity.id)
```

Comment:
If you're going to do it like that, then it's probably best you save directly to an `initial_deck` list. There's no situation in which we care about the rest.